### PR TITLE
cluster_setup_debian: add a branch verification

### PR DESCRIPTION
**Pull request by Paullgk**
Add a task to verify we are configuring the target with a compatible Ansible git branch.  #434

### CODE REVIEW

File: playbooks/cluster_setup_debian.yaml [hunk 1]
```diff
@@ -6,6 +6,21 @@
 # with an iso created by build_debian_iso
 # This playbooks can also configure VMs created with build_debian_iso
 
+- name: Check if we are using a correct seapath flavor
+  hosts:
+    - cluster_machines
+    - standalone_machine
```
Comment (eroussy) [inline playbooks/cluster_setup_debian.yaml:12]: Add VMs in hosts

File: playbooks/cluster_setup_debian.yaml [hunk 2]
```diff
@@ -6,6 +6,21 @@
...
+  tasks:
+    - name: Retrieve Seapath system version
+      command: "uname -a"
+      register: "uname_output"
+
+    - name: Check if current Ansible branch is compatible
+      fail:
+        msg: "You are trying to configure with a Seapath flavor incompatible with the current Ansible git branch"
```
Comment (eroussy) [inline playbooks/cluster_setup_debian.yaml:20]: I would be more straight forward in the message :
"The current branch (debian-main) is incompatible with the Yocto version of SEAPATH. Please use the main branch to configure this hypervisor"

**Review by eroussy — CHANGES_REQUESTED** [review]

File: playbooks/cluster_setup_debian.yaml (hunk 1 above)
Comment (Paullgk) [reply playbooks/cluster_setup_debian.yaml:12]: Done

File: playbooks/cluster_setup_debian.yaml (hunk 2 above)
Comment (Paullgk) [reply playbooks/cluster_setup_debian.yaml]: Done

**Review by eroussy — APPROVED** [review]

**Pull request merged**